src/lib/controller/message_handler.ts

```diff
@@ -132,16 +133,40 @@ export class ControllerMessageHandler {
         );
         return { success };
       }
+      // Schema < 32
       case ControllerCommand.healNode: {
-        const success = await driver.controller.healNode(message.nodeId);
+        const success = await driver.controller.rebuildNodeRoutes(
+          message.nodeId,
+        );
+        return { success };
+      }
+      // Schema >= 32
+      case ControllerCommand.rebuildNodeRoutes: {
```

Comment:
These two cases could be merged, but I guess you'll want to keep them separate in order to be able to add more logic to the 2nd one in the future?

Reply:
I'll combine them for now, we can always split them out later if/when needed